.github/workflows/source-code-scanning.yml

```diff
@@ -0,0 +1,59 @@
+name: Source Code Scanning
+on:
+   schedule:
+     - cron: '0 0 */7 * *'
+   push:
+      # branches:
+      #   - main
+   pull_request:
+      # branches:
+      #   - main
+
+jobs:
+  source-scanning:
+    name: "${{ matrix.DEPLOYMENT_NAME }}  -- Source Code Scanning"
+    runs-on: ubuntu-18.04
+
+    strategy:
+      max-parallel: 5
+      fail-fast: false
+      matrix:
+        DEPLOYMENT_NAME: [ jwebbinar, roman, tike]
+        USE_FROZEN: [ 0 ]
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
I think in some cases (jwebbinar?) we are not using Python 3.8, for package compatibility issues.  Even if that's not the case, the implication here is that the test wouldn't be valid if a deployment uses a different Python version, right?

Reply:
That particular Python-3.8 is just defining the environment which runs bandit and safety.  It is not directly related to environments we install in our images.